scholia/scrape/ojs.py

```diff
@@ -93,12 +93,15 @@ def issue_url_to_paper_urls(url):
         # For instance, for https://tidsskrift.dk/bras/issue/view/9150
         urls = tree.xpath("//h3[@class='media-heading']/a/@href")
     if len(urls) == 0:
-        # For instance, for
-        # https://www.journals.vu.lt/scandinavistica/issue/view/1153
+        # For instance, for https://www.journals.vu.lt/scandinavistica/issue/view/1153
         urls = tree.xpath("//div[@class='article-summary-title']/a/@href")
+    if len(urls) == 0:
+        # Targeting specific class 'summary_title'
+        urls = tree.xpath("//a[@class='summary_title']/@href")
     return urls
 
 
+
```

Comment:
You have an extra line here that results in a styling error.

Please run tox or flake8 to make such that there is no such errors.

Reply:
everything should be good now